modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,300 @@
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score
+
+import multiprocessing as mp
+
+modin.config.NPartitions.put(200)
+
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "type_modin_y",
+    [pd.DataFrame, pd.Series],
+)
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, modin.config.NPartitions.get() + 1],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_breast_cancer(),
+            {"objective": "binary:logistic", "eval_metric": ["logloss", "error"]},
+        ),
+    ],
+    ids=[
+        "load_breast_cancer",
+    ],
+)
+def test_dmatrix_train_predict_binary_classification_datasets(
+    data, num_actors, num_round, type_modin_y
+):
+    dataset, param = data
+
+    X = dataset.data
+    y = dataset.target
+    dmatrix_xgb = xgboost.DMatrix(X, label=y)
+    with pytest.raises(AssertionError):
+        dmatrix_mxgb = xgb.DMatrix(X, label=y)
+
+    modin_X = pd.DataFrame(X)
+    modin_y = type_modin_y(y)
+    dmatrix_mxgb = xgb.DMatrix(modin_X, label=modin_y)
+
+    evals_result_xgb = {}
+    evals_result_mxgb = {}
+    bst = xgboost.train(
+        param,
+        dmatrix_xgb,
+        num_round,
+        evals_result=evals_result_xgb,
+        evals=[(dmatrix_xgb, "train")],
+    )
+    modin_bst = xgb.train(
+        param,
+        dmatrix_mxgb,
+        num_round,
+        evals_result=evals_result_mxgb,
+        evals=[(dmatrix_mxgb, "train")],
+        num_actors=num_actors,
+    )
+
+    predictions = bst.predict(dmatrix_xgb)
+    modin_predictions = modin_bst.predict(dmatrix_mxgb, num_actors=num_actors)
+
+    best_preds = predictions
+    modin_best_preds = modin_predictions
+
+    preds = pd.DataFrame(best_preds).apply(lambda x: round(x))
+    modin_preds = modin_best_preds.apply(lambda x: round(x))
+
+    val = accuracy_score(y, preds)
+    modin_val = accuracy_score(modin_y, modin_preds)
+
+    np.testing.assert_allclose(val, modin_val, atol=0.0018, rtol=0.0018)
+
+
+@pytest.mark.parametrize(
+    "type_modin_y",
+    [pd.DataFrame, pd.Series],
+)
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, 400],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_iris(),
+            {
+                "max_depth": 3,
+                "eta": 0.3,
+                "objective": "multi:softprob",
```

Comment:
```suggestion
                "objective": "multi:softprob",
                "eval_metric": "mlogloss",
```
To exclude xgboost warning during training.

Reply:
This line was added